doc/ref_arch/openstack/chapters/chapter01.rst

```diff
@@ -1,167 +1,167 @@
-are covered in Chapter 5. Chapter 6 covers the implementation and
-enforcement of security capabilities and controls. Life Cycle Management
-of the Cloud Infrastructure and VIM are covered in Chapter 7 with stress
-on Logging, Monitoring and Analytics (LMA), configuration management and
-some other operational items. Please note that Chapter 7 is not a
-replacement for the implementation, configuration and operational
-documentation that accompanies the different OpenStack distributions.
-Chapter 8 identifies certain Gaps that currently exist and plans on how
-to address them (for example, resources autoscaling).
-
-Terminology
------------
-
-General terminology definitions can be found in
-:doc:`cntt:common/glossary` and specific terms relating
-to this reference architecture are to be found in OpenStack Related
-Terminology :ref:`cntt:common/glossary:openstack related terminology`.
-
-Conventions
------------
-
-The key words "must", "must not", "required", "shall", "shall not",
-"should", "should not", "recommended", "may", and "optional"
-in this document are to be interpreted as described in
-RFC 2119 :cite:p:`rfc2119`.
-
-References
-----------
-
-.. bibliography::
-   :cited:
+Introduction
+============
+
+Overview
+--------
+
+This Reference Architecture is focussed on OpenStack as the Virtualised
+Infrastructure Manager (VIM) chosen based on the criteria laid out in
+the :doc:`ref_model:chapters/chapter01`.
+OpenStack :cite:p:`openstack` has the advantage of being a
+mature and widely accepted open-source technology; a strong ecosystem of
+vendors that support it, the OpenInfra Foundation for managing the
+community, and, most importantly, it is widely deployed by the global
+operator community for both internal infrastructure and external facing
+products and services. This means that the operators have existing staff
+with the right skill sets to support a Cloud Infrastructure
+(or Network Function Virtualisation Infrastructure, 
+NFVI :cite:p:`etsinfvinf`)
+deployment into development, test and production. Another reason to
+choose OpenStack is that it has a large active community of vendors and
+operators, which means that any code or component changes needed to
+support the Common Telco Cloud Infrastructure requirements can be
+managed through the existing project communities’ processes to add and
+validate the required features through well-established mechanisms.
+
+Vision
+~~~~~~
+
+The OpenStack-based Reference Architecture will host NFV
+workloads, primarily VNFs (Virtual Network Functions),
+of interest to the Anuket community. The
+Reference Architecture document can be used by operators to deploy
+Anuket conformant infrastructure; hereafter, “conformant” denotes that
+the resource can satisfy tests conducted to verify conformance with this
+reference architecture.
+
+Use Cases
+---------
+
+Several NFV use cases are documented in OpenStack. For more examples and
+details refer to the 

```

Comment:
There's a trailing whitespace here. This line and the next would probably fit into one.

Reply:
@ildikov Thanks! Trailing white space removed